scripts/index.js

```diff
@@ -16,6 +16,26 @@ console.log(isEven(myNums));
 
 function squareNumbers(nums){
     let results = [];
-    nums.forEach();
+    nums.forEach(function(num){
```

Comment:
Consider making a separate commit for the two different solutions (`squareNumbers` and `CitiesTemp`